lib/geocoder.rb

```diff
@@ -0,0 +1,23 @@
+class Geocoder
+  URL = 'rpc.geocoder.us/service/json'
+
+  def self.geocode(params)
+    parse_response(get_json(build_url(params)))
+  end
+
+  private
+
+  def self.parse_response(response)
+    # TODO: handle multiple responses?
+    response.first.symbolize_keys
```

Comment:
I'm not worried about multiple responses; we can just grab the first, which is probably the best. I am, however, worried that we're not catching the Error 500 thrown if the address isn't a match, e.g. http://rpc.geocoder.us/service/json?address=123%20Abc%20Street,%20Middle%20of%20Nowhere,%20AR%2012345 -- we might also have to worry about getting an empty array back, so response.first might be nil, though I haven't seen that happen yet.